src/pyenphase/envoy.py

```diff
@@ -307,7 +305,7 @@ async def request(
         endpoint: str,
         data: dict[str, Any] | None = None,
         method: str | None = None,
-    ) -> httpx.Response:
+    ) -> aiohttp.ClientResponse:
         """
         Make a request to the Envoy.
 
```

Comment:
This no longer retries on Json decode error. Lets change the comment below 

(for some reason the review doesn't include the whole original comment)
```suggestion
        """
        Make a request to the Envoy.

        Send GET or POST request to Envoy. Defaults to GET, specify
        data dictionary to perform a POST. Only specify the endpoint
        path in the Envoy, HTTP type and Envoy address is prepended
        to form full URL based on authentication method.

        Request retries on client connection issues or timeouts.
        Will retry up to 4 times or 50 sec elapsed at next try, which
        ever comes first.


```